hist/hist/test/test_tprofile2poly.cxx

```diff
@@ -0,0 +1,201 @@
+#include "gtest/gtest.h"
```

Comment:
Could you order the includes based on their generality.
First go ROOT related includes, then gtest and then STL.